### PR TITLE
Hand improvements

### PR DESCRIPTION
**Pull request by ldehaine**
### Changes
* Override Hand class' \_\_iadd\_\_ (the one that does `a += b`) method according to issue #15 
* Update related tests
* Reformatting with **black**

### Comments
* You don't know black? Here you go: https://github.com/python/black
Black is a python code beautifier
You can install it with `pip install black`
Run it with `black PATH/TO/YOUR/PYTHON/FILE/OR/A/WHOLE/DIRECTORY`
You can force the accepted line length with `-l 80` (default is 88 for some reason)

Resolves Issue #15

### CODE REVIEW

File: src/CardsAPI/Card.py [hunk 1]
```diff
@@ -79,13 +81,13 @@ def value(self, arg):
             self.name = value_to_card_dict[arg]
         self._value = card_to_value_dict[self.name]
 
-    def __radd__(self, other: Union['Card', int]) -> int:
+    def __radd__(self, other: Union["Card", int]) -> int:
```
Comment (Hboni) [inline src/CardsAPI/Card.py]: Is there a rule for the use of  `'` and `"` ? Or it's just your convenient ?

Comment (ldehaine) [reply src/CardsAPI/Card.py]: https://www.python.org/dev/peps/pep-0008/#string-quotes
TLDR: No rule, but try to be consistent within the same file/project. I personally prefer double quotes because you can safely include apostrophes in the string

Comment (Hboni) [reply src/CardsAPI/Card.py]: It's what I though. I used simple quotes for simple words, and use double quotes for sentences. 
Do you think that we can have apostrophes in our class names ?

Comment (ldehaine) [reply src/CardsAPI/Card.py]: In class names we can't have apostrophes, definitely. Actually I wouldn't have changed those to double quotes myself, Black did it when running, I don't mind but if you prefer single quotes I can change it back

Comment (Hboni) [reply src/CardsAPI/Card.py]: For me, it's more consistent to use simple quotes, as used before ;)

Comment (ldehaine) [reply src/CardsAPI/Card.py]: Fixed

**Review by Hboni — APPROVED** [review]
